.github/workflows/pull_request_ci.yml

```
@@ -0,0 +1,35 @@
+name: Pull request checks
+
+on:
+  pull_request:
+
+jobs:
+  check-unique-standard-names:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
```

Sadly `xmllint` doesn't come with the standard runner environment so we'll need to add it here:

```suggestion
 # Install xmllint so tests pass:
      - name: Install xmllint
        run: |
            sudo apt-get update
            sudo apt-get install libxml2-utils
```

---

xmllint was installed on my local environment already, so I forgot about installing it. Thanks.